### ISSUE TITLE
Modifier for protected component mC_scaled

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1878]
**Reported by otter on 12 Jan 2016 22:40 UTC**
Dymola warning message:
Modifier for protected component mC_scaled. This is not legal Modelica.
File: D:/otter/_Subversion/ModelicaProjects/Modelica/trunk/Modelica/Fluid/Examples/TraceSubstances.mo, line 112
Context: Modelica.Fluid.Examples.TraceSubstances.RoomCO2WithControls.volume.mC_scaled
Original declaration of mC_scaled:
File: D:/otter/_Subversion/ModelicaProjects/Modelica/trunk/Modelica/Fluid/Interfaces.mo, line 601
Context: Modelica.Fluid.Interfaces.PartialLumpedVolume.mC_scaled

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1878

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 12 Jan 2016 22:42 UTC**
Fixed in eda021a89ba7b4da3793f72e6c1ab86aa50aca6c:

Removed modifier and changed
```
package Medium=Modelica.Media.Air.MoistAir(extraPropertiesNames={"CO2"}, 
               C_nominal={1.519E-3});
```
to
```
package Medium=Modelica.Media.Air.MoistAir(extraPropertiesNames={"CO2"}, 
               C_nominal={1.519E-1});
```

**Comment by modelica-trac-importer** [comment]
**Changelog modified by otter on 12 Jan 2016 22:42 UTC**
TraceSubstances.RoomCO2WithControls: Corrected protected modifier
